fuzzers/037-iob-pips/generate.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+from prjxray.segmaker import Segmaker
+import os, re
+import os.path
+
+
+def bitfilter(frame, word):
+    if frame < 28:
```

Comment:
I believe this is too restrictive.  When looking into remaining bits, I believe this should be "if frame > 26".  Try it out.

Reply:
Ok, you mean `if frame < 26` I imagine, right?

Reply:
Yes

Reply:
With that I get the following bits for CLKB:
```
IOI3.IOI_ILOGIC0_CLKB.IOI_IMUX20_1 26_71 27_70 27_98 30_80 31_67 31_77
IOI3.IOI_ILOGIC0_CLKB.IOI_IOCLK0 26_71 27_70 27_98 30_76 30_78 31_67
IOI3.IOI_ILOGIC0_CLKB.IOI_IOCLK1 26_71 27_70 27_98 30_76 30_80 31_67
IOI3.IOI_ILOGIC0_CLKB.IOI_IOCLK2 26_71 27_70 27_98 31_67 31_77 31_79
IOI3.IOI_ILOGIC0_CLKB.IOI_IOCLK3 26_71 27_70 27_98 31_67 31_77 31_81
IOI3.IOI_ILOGIC0_CLKB.IOI_LEAF_GCLK0 26_71 27_70 27_98 30_74 31_67 31_79
IOI3.IOI_ILOGIC0_CLKB.IOI_LEAF_GCLK1 26_71 27_70 27_98 30_74 31_67 31_81
IOI3.IOI_ILOGIC0_CLKB.IOI_LEAF_GCLK2 26_71 27_70 27_98 30_74 30_78 31_67
IOI3.IOI_ILOGIC0_CLKB.IOI_LEAF_GCLK3 26_71 27_70 27_98 30_74 30_80 31_67
IOI3.IOI_ILOGIC0_CLKB.IOI_LEAF_GCLK4 26_71 27_70 27_98 31_67 31_75 31_79
IOI3.IOI_ILOGIC0_CLKB.IOI_LEAF_GCLK5 26_71 27_70 27_98 31_67 31_75 31_81
IOI3.IOI_ILOGIC0_CLKB.IOI_RCLK_FORIO0 26_71 27_70 27_98 30_78 31_67 31_75
IOI3.IOI_ILOGIC0_CLKB.IOI_RCLK_FORIO1 26_71 27_70 27_98 30_80 31_67 31_75
IOI3.IOI_ILOGIC0_CLKB.IOI_RCLK_FORIO2 26_71 27_70 27_98 30_76 31_67 31_79
IOI3.IOI_ILOGIC0_CLKB.IOI_RCLK_FORIO3 26_71 27_70 27_98 30_76 31_67 31_81
```

Where all the 26 and 27 frame address bits are the same and do not change for all of them. I am unsure whether this is correct.

Reply:
Ya, I'm wondering if that is correct.  I guess it could be ILOGIC bits that are correlated with the use of CLKB.  Maybe an improvement for the ILOGIC fuzzer?

Reply:
I will take a look into that, for now I restored the frame check to be `< 28`

Reply:
So, if I allow ILOGIC fuzzer not to generate CLKB, I get a `ISERDESE2.USE` additional bit which is `27_70` for bottom ilogic and `26_57` for top ilogic.

`27_98` is instead related to `IOB33.IOB_Y0.IFF.DDR_CLK_EDGE.OPPOSITE_EDGE` and is produced in the ILOGIC fuzzer. I am quite confused on why they do show up only in the CLKB related pips though.

I think that, at least, the 26 and 27 frame address bits are not related to IOI3 pips.